dosing_pump_controller/ComingSoonMenuItem.cpp

```diff
@@ -0,0 +1,18 @@
+//  Created by Andrew Florjancic on 10/12/24.
+//  Copyright © 2024 Andrew Florjancic. All rights reserved.
+#include "ComingSoonMenuItem.h"
+
+ComingSoonMenuItem::ComingSoonMenuItem(String title) : MenuItem(title, false) {}
+
+void ComingSoonMenuItem::showFeature() {
+    display->lcd.clear();
+    display->lcd.print(display_message);
+}
+
+void ComingSoonMenuItem::presentableWillDismiss() {}
```

Comment:
We should provide a default implementation for these empty methods to reduce clutter, this can be handled in a separate PR.